CHANGELOG.rst

```diff
@@ -6,6 +6,75 @@ Changelog
 
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
+
+3.2.0-alpha.1 (2025-05-23)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+.. warning::
+
+   The Camunda registration backend will be removed in Open Forms 4.0. There is no
+   replacement scheduled - if you rely on this plugin, please get in touch.
+
+.. warning::
+
+    For the generic JSON registration plugin, we changed the way that the data is generated.
+    In case of key conflicts between static, component, and user-defined variables,
+    the static variables will take precedence. Previously, the component and user-defined
+    variables would override the static variables. Our validation guards against the use of
+    keys that are already present in the static variables, but this does not cover old forms
+    and newly-added static variables.
+
+.. warning:: Manual intervention required
+
+    In the 3.1.1 bugfix release we fixed a bug regarding the default values of some components
+    being ``null``. We added a script to fix any forms that still might be affected by these
+    issues. You should run this script after deploying the patch release, to make sure the
+    default values of affected components are fixed.
+
+    .. code-block:: bash
+
+        # in the container via ``docker exec`` or ``kubectl exec``:
+        python /app/bin/fix_component_default_values.py
```

Comment:
is this still relevant here? It specifically mentions "the patch release" but this is an alpha release :)